src/x_gui.c

```diff
@@ -497,6 +497,17 @@ static void pdcontrol_version(t_pdcontrol *x)
     outlet_list(x->x_outlet, &s_list, 3, at);
 }
 
+static void pdcontrol_dollarzero(t_pdcontrol *x, t_symbol *s, t_floatarg f)
+{
+    t_canvas *c = x->x_canvas;
+    for (int i = 0; i < (int)f; i++)
+    {
+        if (c->gl_owner)  /* back up one more into an owner if any */
+            c = c->gl_owner;
+    }
+    outlet_symbol(x->x_outlet, canvas_realizedollar(c, s));
+}
```

Comment:
this will realize *all* dollargs, not just dollarzero.

so the method name `dollarzero` is plainly wrong.

Reply:
I guess the best way out is to rename it then, so I did change it to "realizedollar"